tools/releaseBuild/Images/GenericLinuxFiles/PowerShellPackage.ps1

```diff
@@ -25,33 +27,26 @@ try {
     Set-Location $location
     Import-Module "$location/build.psm1"
     Import-Module "$location/tools/packaging"
-    
+
     Start-PSBootstrap -Package -NoSudo
     Start-PSBuild -Crossgen -PSModuleRestore @releaseTagParam
 
     Start-PSPackage @releaseTagParam
-    if($AppImage.IsPresent)
+    switch ($ExtraPackage)
     {
-        Start-PSPackage -Type AppImage @releaseTagParam
+        "AppImage" { Start-PSPackage -Type AppImage @releaseTagParam }
+        "tar"      { Start-PSPackage -Type tar @releaseTagParam }
     }
 }
 finally
 {
     Pop-Location
 }
 
-$linuxPackages = Get-ChildItem "$location/powershell*" -Include *.deb,*.rpm
-    
-foreach($linuxPackage in $linuxPackages) 
-{ 
-    Copy-Item -Path $linuxPackage.FullName -Destination $destination -force
-}
-
-if($AppImage.IsPresent)
+$linuxPackages = Get-ChildItem "$location/powershell*" -Include *.deb,*.rpm,*.AppImage,*.tar.gz
```

Comment:
In the previous commit, *.deb and *.rpm packages are found under $location/powershell* while *.AppImage and *.tar.gz are found under $location. With this change *.AppImage and *.tar.gz package won't be found.

Reply:
It's not under `$location/powershell`, but `$location/powershell*` (there is a wildcard * here). All packages start with `powershell`.